RFS/src/main/java/org/opensearch/migrations/bulkload/common/OpenSearchClient.java

```diff
@@ -44,7 +44,7 @@ public class OpenSearchClient {
             .maxBackoff(DEFAULT_MAX_BACKOFF)
             .filter(throwable -> !(throwable instanceof InvalidResponse)); // Do not retry on this exception
 
-    private static final int BULK_MAX_RETRY_ATTEMPTS = 15;
+    private static final int BULK_MAX_RETRY_ATTEMPTS = 2;
```

Comment:
Oooh, this is a potentially really impactful change.  Curious what issue you ran into that led you to this?

Reply:
! Thanks for catching this, I was debugging failing requests and didn't want to wait the extra minute.